pkg/adaptor/cloud/ibmcloud-powervs/manager.go

```diff
@@ -0,0 +1,37 @@
+// (C) Copyright Confidential Containers Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package ibmcloud_powervs
+
+import (
+	"flag"
+
+	"github.com/confidential-containers/cloud-api-adaptor/pkg/adaptor/cloud"
+)
+
+var ibmcloudPowerVSConfig Config
+
+type Manager struct{}
+
+func (_ *Manager) ParseCmd(flags *flag.FlagSet) {
+
+	flags.StringVar(&ibmcloudPowerVSConfig.ApiKey, "api-key", "", "IBM Cloud API key, defaults to `IBMCLOUD_API_KEY`")
+	flags.StringVar(&ibmcloudPowerVSConfig.Zone, "zone", "", "PowerVS zone name")
+	flags.StringVar(&ibmcloudPowerVSConfig.ServiceInstanceID, "service-instance-id", "", "ID of the PowerVS Service Instance")
+	flags.StringVar(&ibmcloudPowerVSConfig.NetworkID, "network-id", "", "ID of the network instance")
+	flags.StringVar(&ibmcloudPowerVSConfig.ImageID, "image-id", "", "ID of the boot image")
+	flags.StringVar(&ibmcloudPowerVSConfig.SSHKey, "ssh-key", "", "Name of the SSH Key")
+	flags.Float64Var(&ibmcloudPowerVSConfig.Memory, "memory", 2, "Amount of memory in GB")
+	flags.Float64Var(&ibmcloudPowerVSConfig.Processors, "cpu", 0.5, "Number of processors allocated")
```

Comment:
Is there any instance size or flavour concept yet in powervs?

Reply:
@bpradipt, not yet.